lib/index.ts

```diff
@@ -193,8 +191,8 @@ async function runContentfulImport (params: RunContentfulImportParams) {
       })
     })
     .then((data) => {
-      const errorLog = log.filter((logMessage) => logMessage.level !== 'info' && logMessage.level !== 'warning')
-      const displayLog = log.filter((logMessage) => logMessage.level !== 'info')
+      const errorLog = log.filter(isErrorLog)
+      const displayLog = log.filter(isDisplayLog)
```

Comment:
`log` only accepts messages with levels `error` or `warn`, I think this filter does nothing?